testRun.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SCRIPT_PATH="./exokit.sh"
+
+# Execute ./exokit.sh -h and log the console output to testRunLog.txt,
+# stdout and stderr included
+"$SCRIPT_PATH" -h >testRunLog.txt 2>&1 &
+echo "Exokit loaded, Exit Code: " $?
+
+sleep 5s
```

Comment:
Instead of this, we could just make exokit have a flag that makes it render once and then exit success or failure depending on whether it thinks it drew a frame.